typings/index.d.ts

```diff
@@ -114,47 +111,53 @@ interface OptionalExtraGetItemPropsOptions {
   [key: string]: any
 }
 
-export interface GetItemPropsOptions extends OptionalExtraGetItemPropsOptions {
+export interface GetItemPropsOptions<Item>
+  extends OptionalExtraGetItemPropsOptions {
   index?: number
-  item: any
+  item: Item
 }
 
-export interface PropGetters {
+export interface PropGetters<Item> {
   getRootProps: (options: GetRootPropsOptions) => any
   getButtonProps: (options?: GetButtonPropsOptions) => any
   getLabelProps: (options?: GetLabelPropsOptions) => any
   getInputProps: (options?: GetInputPropsOptions) => any
-  getItemProps: (options: GetItemPropsOptions) => any
+  getItemProps: (options: GetItemPropsOptions<Item>) => any
 }
 
-export interface Actions {
-  openMenu: (cb?: Function) => void
-  closeMenu: (cb?: Function) => void
-  toggleMenu: (cb?: Function) => void
-  selectItem: (item: any, otherStateToSet?: object, cb?: Function) => void
+export interface Actions<Item> {
+  openMenu: (cb?: Callback) => void
+  closeMenu: (cb?: Callback) => void
+  toggleMenu: (cb?: Callback) => void
+  selectItem: (item: Item, otherStateToSet?: {}, cb?: Callback) => void
   selectItemAtIndex: (
     index: number,
-    otherStateToSet?: object,
-    cb?: Function,
+    otherStateToSet?: {},
+    cb?: Callback,
   ) => void
-  selectHighlightedItem: (otherStateToSet?: object, cb?: Function) => void
+  selectHighlightedItem: (otherStateToSet?: {}, cb?: Callback) => void
   setHighlightedIndex: (
     index: number,
-    otherStateToSet?: object,
-    cb?: Function,
+    otherStateToSet?: {},
+    cb?: Callback,
   ) => void
-  clearSelection: (cb?: Function) => void
+  clearSelection: (cb?: Callback) => void
   clearItems: () => void
-  reset: (otherStateToSet?: object, cb?: Function) => void
-  itemToString: (item: any) => string
+  reset: (otherStateToSet?: {}, cb?: Callback) => void
```

Comment:
Replaced `object` with `{}` and `Function` with `Callback`

Reply:
Does `{}` allow to put a non-empty object to the argument?

Reply:
Yes https://www.typescriptlang.org/play/#src=const%20i%20%3D%20(x%3A%20%7B%7D)%20%3D%3E%20x%0D%0Ai(%7B%20a%3A%201%20%7D)%0D%0A